api/Comment.json

```diff
@@ -77,10 +77,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": "7"
```

Comment:
I thought this might be wrong, but confirmed by testing `new Comment('a')` on Safari 6 manually.